test/.tarantoolctl

```diff
@@ -0,0 +1,14 @@
+-- Options for test-run tarantoolctl
+local workdir = os.getenv('TEST_WORKDIR')
+default_cfg = {
+    pid_file   = workdir,
+    wal_dir    = workdir,
+    snap_dir   = workdir,
+    vinyl_dir  = workdir,
+    logger     = workdir,
+    background = false,
+}
+
+instance_dir = workdir
+
+-- vim: set ft=lua :
```

Comment:
I would prefer to use the tarantoolctl config from test-run. See https://github.com/tarantool/tarantool/commit/6c8efa8491ed3fc2834938ec65d91dc1f7ec7e34 and https://github.com/tarantool/tarantool/commit/54667f6fa86ff1ab0030501abc9a59d0d7aea82f. Is there a reason to place our own config here?

(I would also highlight that `logger` and `snap_dir` options are [deprecated](https://www.tarantool.io/en/doc/latest/reference/configuration/#deprecated-parameters).)

Reply:
There are no reason for this.
Ok, removed custom config to use `.tarantoolctl` in a root dir.